xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/dto/MarginMode.java

```diff
@@ -1,6 +1,28 @@
 package org.knowm.xchange.okcoin.v3.dto;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
 public enum MarginMode {
   crossed,
-  fixed;
+  fixed,
+  empty // special case, for some wallets okex returns empty margin mode
+;
+
+  @JsonCreator
+  public static MarginMode create(String s) {
+    if (s == null) {
+      return null;
+    }
+    if (s.isEmpty()) {
+      return empty;
```

Comment:
wouldn't it be better to just return null in this case also?

Reply:
sure, we could use null here. But what if you need to differ between null and empty?
was not sure and decided to keep empty type in place